File: ruby/sum.rb
even = []
odd = []
num = 0

while num < 5 do

input = gets.to_i

  if input % 2 == 0
    even << input
  else
    odd << input
  end
  
  num += 1
end

puts "偶数の合計：#{even.sum}"

puts "奇数の合計：#{odd.sum}"

File: .ipynb_checkpoints/python/sum.py

even = []
odd = []
num = 0

while num < 5:
  text1 = "{}番目の数字を入力"
  
  print(text1.format(num + 1))
  
  value = int(input())
  
  num = num + 1
  
  print()
  
  if value % 2 == 0:
    even.append(value)
  else:
    odd.append(value)
    
text2 = "偶数の合計：{}"
text3 = "奇数の合計：{}"

print(text2.format(sum(even)))
print(text3.format(sum(odd)))

